cmd/insights/commands/upload.go

```diff
@@ -40,23 +42,45 @@ func installUploadCmd(app *App) {
 	uploadCmd.Flags().UintVar(&app.config.Upload.MinAge, "min-age", defaultMinAge, "the minimum age (in seconds) of a report before the uploader will attempt to upload it")
 	uploadCmd.Flags().BoolVarP(&app.config.Upload.Force, "force", "f", false, "force an upload, ignoring min age and clashes between the collected file and a file in the uploaded folder, replacing the clashing uploaded report if it exists")
 	uploadCmd.Flags().BoolVarP(&app.config.Upload.DryRun, "dry-run", "d", false, "go through the motions of doing an upload, but do not communicate with the server or send the payload")
+	uploadCmd.Flags().BoolVarP(&app.config.Upload.BackoffRetry, "backoff-retry", "b", false, "enable exponential backoff retry for failed uploads")
```

Comment:
I'd name this parameter just as  "retry" and leave the fact that's exponential backoff as an implementation detail that we reserve the rights to change at any point in time if we see fit.